### ISSUE TITLE
Reverse for 'app_list' with keyword arguments '{'app_label': 'filebrowser'}' not found

### ISSUE THREAD

**Issue by leogout** [smacker/django-filebrowser-no-grappelli#64]
Hello, I am having trouble installing django-filebrowser-no-grappelli.
I've followed the steps to install it but if I go to `http://127.0.0.1:8000/admin/filebrowser/browse/`, I get the following error :

```
NoReverseMatch at /admin/filebrowser/browse/
Reverse for 'app_list' with keyword arguments '{'app_label': 'filebrowser'}' not found. 1 pattern(s) tried: ['admin/(?P<app_label>store|taggit)/$']
```
I've been looking for a solution for hours but I am unable to understand this error...
I am using Django 2.2.2 and django-filebrowser-no-grappelli 3.8.0.
Sorry if this is the wrong place to ask, feel free to redirect my request somewhere else.

**Comment by leogout** [comment]
Haha I was trying again to install this lib in another project... What a surprise to come up to my own issue by typing this problem in Google !

Any input would be very appreciated.

**Comment by leogout** [comment]
All right I got tired of this, here is an alternative which have the file upload out of the box: https://django-ckeditor.readthedocs.io/

**Comment by christianwgd** [comment]
Hey, leogut,

if you wanna give it another try, you can have a look at my minimal filebrowser sample **[https://github.com/christianwgd/filebrowser_sample](url)**

I was curious if i missed some changes cause i used this in several projects. Found that everything works as expected. My guess is, you didn't inlcude `path('admin/filebrowser/', site.urls),` in your settings file or didn't put in first place.

I don't know what status this project has. @smacker's last changes were more then one year ago and i'm also waiting for a fix for Django > 3.1.1.

**Comment by smacker** [comment]
Hi @leogout as @christianwgd correctly pointed out you forgot to add the path to filebrowser to your URLs config.
